torch/nn/_intrinsic/qat/modules/conv_fused.py

```diff
@@ -129,8 +127,8 @@ def _forward(self, input):
             conv = conv * rescale_factor.reshape([1, -1, 1, 1])
             conv = conv + (self.beta - self.gamma * batch_mean * batch_rstd).reshape([1, -1, 1, 1])
 
-            self.running_mean = exponential_average_factor * batch_mean + (1 - exponential_average_factor) * self.running_mean
-            self.running_var = exponential_average_factor * batch_var + (1 - exponential_average_factor) * self.running_var
+            self.running_mean = exponential_average_factor * batch_mean.detach() + (1 - exponential_average_factor) * self.running_mean
```

Comment:
@ezyang just want to make sure these detach() are used correctly.

Reply:
LGTM